src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php

```diff
@@ -69,11 +73,13 @@ final class LogGridDefinitionFactory extends AbstractGridDefinitionFactory
     public function __construct(
         HookDispatcherInterface $hookDispatcher,
         $resetActionUrl,
-        $redirectionUrl
+        $redirectionUrl,
+        $isMultistoreFeatureEnabled
```

Comment:
BC Break => to document

Reply:
🤔 This was injected into the service. See https://github.com/PululuK/PrestaShop/blob/patch-7/src/PrestaShopBundle/Resources/config/services/core/grid/grid_definition_factory.yml#L19

Reply:
I agree, but it's a BC Break, because you broke the existing signature of the method.

Imagine a contributor which use this class. If it's not documented in release notes, he will cry :sob: .

Reply:
Where I should write the documentation? 
Could you give me a concrete example please?
You are doing a great job here https://github.com/PrestaShop/PrestaShop/pull/20118, But I can't find this doc example.

Reply:
Done ! Thanks @Progi1984